tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/commons/js/dialog/export-dialog.js

```diff
@@ -182,46 +190,46 @@ define(['require', 'jquery', 'log', 'backbone', 'smart_wizard', 'siddhiAppSelect
                                 };
                                 self._fill_template_value_dialog = new FillTemplateValueDialog(fillTemplateOptions);
                                 self._fill_template_value_dialog.render();
                             } else if (stepNumber === 5) {
-                                self.kubernetesConfigModel = new KubernetesConfigDialog({
-                                    app: self.app,
+                                self._kubernetesConfigModel = new KubernetesConfigDialog({
+                                    app: self._options,
                                     templateHeader: exportContainer.find('#kubernetes-configuration-step-id')
                                 });
-                                self.kubernetesConfigModel.render();
+                                self._kubernetesConfigModel.render();
                             }
                         }
                     });
 
-                    this.exportContainer = exportContainer;
+                    this._exportContainer = exportContainer;
                 },
 
                 sendExportRequest: function () {
-                    var self = this;
-                    var type;
-                    if (this.isDocker) {
-                        type = 'docker';
-                    } else {
-                        type = 'kubernetes';
-                        this.payload.kubernetesConfiguration = this.kubernetesConfigModel.getKubernetesConfigs();
+                    if (!this._isExportDockerFlow) {
+                        this._payload.kubernetesConfiguration = this._kubernetesConfigModel.getKubernetesConfigs();
                     }
-                    this.payload.bundles = this.jarsSelectorDialog.getSelected('bundles');
-                    this.payload.jars = this.jarsSelectorDialog.getSelected('jars');
+                    this._payload.bundles = this._jarsSelectorDialog.getSelected('bundles');
+                    this._payload.jars = this._jarsSelectorDialog.getSelected('jars');
 
-                    var payload = $('<input id="payload" name="payload" type="text" style="display: none;"/>')
-                        .attr('value', JSON.stringify(this.payload));
+                    var payloadInputField = $('<input id="payload" name="payload" type="text" style="display: none;"/>')
+                        .attr('value', JSON.stringify(this._payload));
+                    this._btnExportForm.append(payloadInputField);
 
-                    var exportUrl = this.app.config.baseUrl + "/export?type=" + type;
-
-                    this.btnExportForm.append(payload);
-                    this.btnExportForm.attr('action', exportUrl);
-
-                    $(document.body).append(this.btnExportForm);
-                    this.btnExportForm.submit();
+                    $(document.body).append(this._btnExportForm);
+                    this._btnExportForm.submit();
+                    // todo handle error
```

Comment:
remove todos

Reply:
Open an issue for todo https://github.com/siddhi-io/distribution/issues/377